restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -102,7 +106,8 @@
     allColumnTypes.add(Type.Boolean);
     allColumns.add(Column.create("bool_value", Type.Boolean));
 
-    forbiddenCharacters = ImmutableList.of('[', ']', ',', '.', '\'', '*');
+    escapableCharacters = ImmutableList.of('[', ']', ',', '.', '\'', '*');
+    forbiddenCharacters = ImmutableList.of('%');
```

Comment:
Why  do we forbid `%` chars? The URL encoding scheme should be able to support escaping them, I belive.

Reply:
Yeah, it should be fine for escaping to work at multiple levels I think? But I could be missing something.